new_jersey/ver2/style.css

```diff
@@ -0,0 +1,171 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+@font-face {
+    font-family: 'Proxima Nova Light';
+    src: url('fonts/Mark Simonson - Proxima Nova Light.eot');
+    src: local('☺'), url('fonts/Mark Simonson - Proxima Nova Light.woff') format('woff'), url('fonts/Mark Simonson - Proxima Nova Light.ttf') format('truetype'), url('fonts/Mark Simonson - Proxima Nova Light.svg') format('svg');
+    font-weight: normal;
+    font-style: normal;
+}
+@font-face {
+    font-family: 'SourceSansPro-Light';
+    src: url('fonts/SourceSansPro-Light.eot');
+    src: local('☺'), url('fonts/SourceSansPro-Light.woff') format('woff'), url('fonts/SourceSansPro-Light.ttf') format('truetype'), url('fonts/SourceSansPro-Light.svg') format('svg');
+    font-weight: normal;
+    font-style: normal;
+}
+@font-face {
+    font-family: 'SourceSansPro-Semibold';
+    src: url('fonts/SourceSansPro-Semibold.eot');
+    src: local('☺'), url('fonts/SourceSansPro-Semibold.woff') format('woff'), url('fonts/SourceSansPro-Semibold.ttf') format('truetype'), url('fonts/SourceSansPro-Semibold.svg') format('svg');
+    font-weight: normal;
+    font-style: normal;
+}
+@font-face {
+    font-family: 'Proxima Nova Bold';
+    src: url('fonts/Mark Simonson - Proxima Nova Bold.eot');
+    src: local('☺'), url('fonts/Mark Simonson - Proxima Nova Bold.woff') format('woff'), url('fonts/Mark Simonson - Proxima Nova Bold.ttf') format('truetype'), url('fonts/Mark Simonson - Proxima Nova Bold.svg') format('svg');
+    font-weight: normal;
+    font-style: normal;
+}
+
+.container {
+    background-color: #4e5860;
+    width: 800px;
```

Comment:
The idea behind the mock-up was that the gray background fills up the whole screen, and the widget is centered on it. So I would suggest changing the .container width and height to 100vw and 100 vh accordingly.